shopsdk/README.md

```diff
@@ -0,0 +1,76 @@
+# TODO:<br>
+```
+Import the dynamic config into the code
+Write stickers script
+```
+To setup the purely configurable enviroment 6100m made, simply do this command.<br>
+```
+python3 setup.py 0
+```
+Be sure to follow all prompts.<br>
+
+There are a few services these scripts provide that you can offer on the shop!<br>
+The services it currently provides:<br>
+1. My Aquarium Mystery DLC Tanks<br>
+2. iOS 1.0 - 1.1.2 Jailbreak Server<br>
+
+To run the My Aquarium DLC script, do:<br>
+```
+python3 install.py 9 0 0 0 0 0 0 0 0
```

Comment:
What do the arguments mean here? Please attach documentation.

Reply:
Actually, those 0's are typically different because they are mode specific to other modes, but this mode just nullifies them because it doesn't need em :)